istio-cni/templates/clusterrolebinding.yaml

```diff
@@ -13,3 +13,19 @@ subjects:
 - kind: ServiceAccount
   name: istio-cni
   namespace: {{ .Release.Namespace }}
+---
+{{- if hasKey .Values.cni "psp_cluster_role" }}
```

Comment:
Not sure what this is, how it'll be used, what the cluster role will be, etc.

Please add a wiki page or some inline docs.

Reply:
Added some docs in the `values.yaml`